packages/core/scripts/identifier-umip/1_Propose.js

```diff
@@ -2,13 +2,16 @@
 // fork of the main net or can be run directly on the main net to execute the upgrade transactions.
 // To run this on the localhost first fork main net into Ganache with the proposerWallet unlocked as follows:
 // ganache-cli --fork https://mainnet.infura.io/v3/d70106f59aef456c9e5bfbb0c2cc7164 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25
-// Then execute the script as: truffle exec ./scripts/identifier-umip/1_Propose.js --network mainnet-fork --identifier USDETH --identifier ETHBTC from core
+// Then execute the script as: yarn truffle exec ./scripts/identifier-umip/1_Propose.js --network mainnet-fork --identifier USDETH --identifier ETHBTC from core
 
 // Use the same ABI's as deployed contracts:
 const { getTruffleContract } = require("../../index");
 const Governor = getTruffleContract("Governor", web3, "1.1.0");
 const IdentifierWhitelist = getTruffleContract("IdentifierWhitelist", web3, "1.1.0");
+const Finder = artifacts.require("Finder");
+const Voting = artifacts.require("Voting");
```

Comment:
nit: prefer the `getTruffleContract` syntax so we can explicitly specify the version and we don't depend on truffle's `artifacts.require`, which means we would need to run this inside of a truffle exec script.

It's also probably a good idea to bump the versions of all of these to "latest".